content/blog/2016-05-09-when-to-use-varbit-chunks.md

```diff
@@ -0,0 +1,159 @@
+---
+title: When (not) to use varbit chunks
+created_at: 2016-05-09
```

Comment:
Would prefer tomorrow. Simplifies my job... and i really like the CNCF post to be on top of feeds etc for more than a day.